spec/item_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './item'
+require './label'
+require './genre'
+require './author'
+
+describe Item do
+  item1 = Item.new('20/10/2020')
+
+  describe 'if a parameter passed correctly' do
+    it 'should return the correct parameter' do
+      expect(item1.publish_date).to eq '20/10/2020'
+    end
+  end
+
+  describe '#add_label' do
+    it 'should add the label correctly' do
+      label1 = Label.new('Education', 'Blue')
+      item1.add_label(label1)
+      expect(item1.label).to be label1
+    end
+  end
+
+  describe '#add_author' do
+    it 'should add the label correctly' do
+      author1 = Author.new('Ellen', 'White')
+      item1.add_author(author1)
+      expect(item1.author).to be author1
+    end
+  end
+
+  describe '#add_genre' do
+    it 'should add the label correctly' do
+      genre1 = Genre.new('Spiritual')
+      item1.add_genre(genre1)
+      expect(item1.genre).to be genre1
+    end
+  end
+end
```

Comment:
- [ ] Well done implementing the tests so far 👏. However, as stated in the requirements, all methods inside classes should be tested. So please ensure to add tests for the `can_be_archived?` and the `move_to_archive` methods from the `Item` class. 👌

### Requirement :point_down: 
![Screenshot from 2023-11-03 14-59-27](https://github.com/dvcorreia1984/ruby-group-capstone/assets/89228743/62dd0196-4c4f-4871-86a0-c8f997697f3b)